_posts/2024-09-20-wasmtime-25.0.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Wasmtime v25.0 brings extended constants, WASI 0.2.1, user stack maps, and more"
+author: "Nick Fitzgerald"
+date: "2024-09-20"
+github_name: "fitzgen"
```

Comment:
I think you probably copy-pasted this bit and want to update this so that you're the author :)